packages/shared/components/popups/StopVotingPopup.svelte

```diff
@@ -31,13 +32,17 @@
         <TextHint info text={localize('popups.stopVoting.hint')} />
     </div>
     <div class="flex w-full space-x-4 mt-6">
-        <Button outline classes="w-full" onClick={onCancelClick}>{localize('actions.cancel')}</Button>
+        <Button outline classes="w-full" disabled={isTransferring} onClick={onCancelClick}
```

Comment:
Same as above ^

Reply:
Because cancel to the user means that it will cancel an action, if you have already kicked off the action then the cancel button is not cancelling anything.

So its confusing to the user if it doesn't do anything. Why would we have a cancel button that only closes a popup when there is a close button.